src/storage/query/QueryBaseProcessor.inl

```diff
@@ -488,6 +488,7 @@ cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkExp(const Expression* exp,
         case Expression::Kind::kUUID:
         case Expression::Kind::kVar:
         case Expression::Kind::kVersionedVar:
+        case Expression::Kind::kPathBuild:
         default: {
```

Comment:
Because there is no special logic to deal with this case, just use default, right?

Reply:
Yes